src/core/unicode.js

```diff
@@ -1640,7 +1640,17 @@ function reverseIfRtl(chars) {
   return buf.join("");
 }
 
+const SpecialCharRegExp = new RegExp("^(\\s)|(\\p{Mn})$", "u");
```

Comment:
*Possibly a dumb question, but:* Is there a reason for using `^` and `$` explicitly, rather than simply using the global flag?

Reply:
According to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match#return_value,
`g` mustn't be used in order to get capturing groups.